Assets/Scripts/WorkPuzzle/SpawnText.cs

```diff
@@ -14,8 +14,7 @@ public class SpawnText : MonoBehaviour
     public WordDisplay Spawn()
     {
         GameObject textObject;
-        //Vector3 position = new Vector3(Random.Range(90f, 900f), 500f);
-        Vector3 position = new Vector3(Random.Range(90f, Screen.width-90f), Screen.height);
+        Vector3 position = new Vector3(Random.Range(90f, Screen.width-80f), Screen.height);
```

Comment:
What are the magic numbers 90f and 80f here? Screen.width is in pixels, so if these are meant to be a static offset this will not work well with large screen resolutions.

Reply:
Good catch, meant to change those too. I will have them changed in next PR!